gateway-ha/src/main/java/com/lyft/data/gateway/ha/persistence/dao/ResourceGroup.java

```diff
@@ -0,0 +1,115 @@
+package com.lyft.data.gateway.ha.persistence.dao;
+
+import static com.lyft.data.gateway.ha.router.PrestoResourceManager.ResourceGroupDetail;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.javalite.activejdbc.Model;
+import org.javalite.activejdbc.annotations.BelongsTo;
+import org.javalite.activejdbc.annotations.Cached;
+import org.javalite.activejdbc.annotations.IdName;
+import org.javalite.activejdbc.annotations.Table;
+
```

Comment:
Can we add some javadocs here explaining where this table is used and why it's added?

Reply:
In the javadocs, can we add a link to the tables schemas in Presto? Since we got the schemas from presto. Please do the same for other dao files too.

Reply:
I added the tables schemas to gateway-ha-persistence.sql. Should I link to that or to here: https://github.com/lyft/presto/blob/master/presto-resource-group-managers/src/test/java/io/prestosql/plugin/resourcegroups/db/H2ResourceGroupsDao.java